docs/cheaha/slurm/slurm_tutorial.md

```diff
@@ -22,13 +26,31 @@ The last portion is running the actual code or software. Here, the computational
+This user guide provides comprehensive insight into different types of batch jobs, facilitating in identifying the most suitable job type for your specific tasks. With clear explanations and practical examples, you will gain a deeper understanding of sequential, parallel, array, multicore, GPU, and multi-node jobs, assisting to make informed decisions when submitting jobs on the Cheaha system.
+
+1. [Simple Batch Job](#example-1-a-simple-slurm-batch-job) is ideal for Cheaha users who are new to Slurm batch job submission.
+
+2. [Sequential Batch Job](#example-2-sequential-job) is for running tasks that are dependent on each other and require only one CPU resource. Increasing the number of CPUs typically does not enhance performance in these cases. For instance, a Python or R script that executes a series of steps—such as data loading, extraction, analysis, and output reporting—where each step must be completed before the next can begin.
+
+3. [Parallel Job](#example-3-parallel-jobs) is suitable for executing multiple independent tasks/jobs simultaneoulsy and efficiently distributing them across resources. This approach is particularly beneficial for small-scale tasks that cannot be split into parallel processes within the code itself. For example, consider a Python script that operates on different data set, in such a scenario, you can utilize srun to execute multiple instances of the script concurrently, each operating on a different dataset and on different resources.
+
+4. [Array Job](#example-4-array-job) is used for submitting and running mutiple large number of identical tasks in parallel. They share the same code and execute with similar resource requirements. For instance, array jobs can be designed for executing multiple instances of the same task with slight variations in inputs or parameters such as perform [FastQC](https://home.cc.umanitoba.ca/~psgendb/doc/fastqc.help) processing on 10 different samples.
 
+5. [Mutlicore Job](#example-5-multithreaded-or-multicore-job) is used when the software itself support multithreaded parallelism inherently. For instance, there are numerous software such as [MATLAB](https://www.mathworks.com/help/matlab/ref/parfor.html), [FEBio](https://help.febio.org/FebioUser/FEBio_um_3-4-Section-2.6.html), [Xplor-NIH](https://nmr.cit.nih.gov/xplor-nih/doc/current/helperPrograms/options.html), that support implicit parallelism and can be run on multiple cores by simply configuring with the appropriate options.
+
+6. [GPU Job](#example-6-gpu-job) are appropriate for pipelines and software that are designed to run on GPU-based systems. Example includes [Tensorflow](https://www.tensorflow.org/guide/gpu), [Parabricks](../../education/case_studies.md), [PyTorch](https://pytorch.org/tutorials/prototype/ios_gpu_workflow.html#prototype-use-ios-gpu-in-pytorch), etc.
```

Comment:
Very briefly explain how GPUs are parallel, i.e., they have many cores specially designed to all do the same math at the same time.